<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>psviewer</class>
 <widget class="QDialog" name="psviewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pusle Sequence Viewer</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>470</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QCustomPlot" name="PSV" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>631</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkZoomY">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>460</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Zoom and drag Y axis</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkZoomX">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>485</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Zoom and drag X axis</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboPlotItem">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>470</y>
     <width>104</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>473</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select signal to plot</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>psviewer</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>psviewer</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
